tests/helpers/common-helpers.psm1

```diff
@@ -4,6 +4,11 @@ if (-not ("NuGet.Versioning.VersionRange" -as [Type])) {
     Add-Type -Path $PSScriptRoot\common\NuGet.Versioning.dll
 }
 
+if ([string]::IsNullOrEmpty($env:ChocolateyInstall)) {
+    $env:ChocolateyInstall = Join-Path $env:ProgramData 'Chocolatey'
+    Write-Warning "ChocolateyInstall Variable was not set. It has been set to $($env:ChocolateyInstall)."
+}
+

```

Comment:
@gep13 this isn't a particularly elegant solution to the issue we were seeing on Test Kitchen, but it does solve the issue seen there.

Reply:
@corbob @vexx32 we need to back out this change, and find the solution to the underlying problem.

Reply:
I've backed this out in a revert commit. We can rebase it away before merge.